src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -5165,15 +5168,42 @@ namespace AirflowNetworkBalanceManager {
 					valsByFacade[ FacadeNum - 1 ][ WindDirNum - 1 ] = 0.6 * std::log( 1.248 - 0.703 * std::sin( IncRad / 2.0 ) - 1.175 * pow_2( std::sin( IncRad ) ) + 0.131 * pow_3( std::sin( 2.0 * IncRad * SideRatioFac ) ) + 0.769 * std::cos( IncRad / 2.0 ) + 0.07 * pow_2( SideRatioFac * std::sin( IncRad / 2.0 ) ) + 0.717 * pow_2( std::cos( IncRad / 2.0 ) ) );
 				} // End of wind direction loop
 			} // End of facade number loop
+			// Add a roof
+			FacadeNum = 5;
+			SR = min(max(SideRatio, 0.25), 1.0);
```

Comment:
Some missing padding in the parentheses; if there is anything functional to change I'll update that as well.